docs/monty.dev.md

```diff
@@ -26,19 +26,6 @@ with a possible replacement.
 * **Returns**
   Original function, but with a warning to use the updated class.
 
-## monty.dev.get_ncpus()
-
-**NOTE**: If you are using Python >= 2.7, multiprocessing.cpu_count() already
-provides the number of CPUs. In fact, this is the first method tried.
-The purpose of this function is to cater to old Python versions that
-still exist on many Linux style clusters.
-
-Number of virtual or physical CPUs on this system, i.e.
-user/real as output by time(1) when called with an optimally scaling
-userspace-only program. Return -1 if ncpus cannot be detected. Taken from:
-[http://stackoverflow.com/questions/1006289/how-to-find-out-the-number-of](http://stackoverflow.com/questions/1006289/how-to-find-out-the-number-of)-
-cpus-in-python
```

Comment:
Is the docs automatically generated? It seems this method doesn't exist anymore.

Reply:
yes, the docs are auto-generated from doc strings. possible that this needs to be rerun @shyuep?

Reply:
Thanks for the input.